presto-main/src/main/java/com/facebook/presto/execution/CreateViewTask.java

```diff
@@ -82,7 +82,7 @@ public String explain(CreateView statement, List<Expression> parameters)
 
         accessControl.checkCanCreateView(session.getRequiredTransactionId(), session.getIdentity(), name);
 
-        String sql = getFormattedSql(statement.getQuery(), sqlParser, Optional.of(parameters));
+        String sql = SqlFormatter.formatSql(statement.getQuery(), Optional.of(parameters));
```

Comment:
@electrum Using SqlFormatter.formatSql will throw exception when there is a quote in table name:
create view test as select * from mysql."schema-name".table limit 10;
Here the quotes are already erased, and schema-name becomes invalid

not sure whether really need the round-trip verification here
SqlQueryManager already catches parsing exception. If really need round-trip verification, maybe we could do something in SqlQueryManager?